backend/tests/integration/test_dashboard_integration.py

```
@@ -389,39 +383,9 @@ async def process_session(sid):
         await asyncio.gather(*tasks)
         
         # Verify sessions were processed (completed ones are archived)
-        all_sessions = update_service.get_all_active_sessions()
+        all_sessions = list(update_service.active_sessions.values())
```

Ruff (0.12.2)

Local variable `all_sessions` is assigned to but never used

Remove assignment to unused variable `all_sessions`

(F841)